pkg/yurtctl/cmd/convert/edgenode.go

```diff
@@ -403,8 +387,8 @@ func (c *ConvertEdgeNodeOptions) ResetKubelet() error {
 	return nil
 }
 
-func (c *ConvertEdgeNodeOptions) getYurthubYaml() string {
-	return filepath.Join(c.PodMainfestPath, enutil.YurthubYamlName)
+func (c *ConvertEdgeNodeOptions) getYurthubYaml(podManifestPath string) string {
```

Comment:
How about remove func `getYurthubYaml()` as this func is not related to `ConvertEdgeNodeOptions`.

Reply:
defining the YAML file path is the responsibility of convert.
so are the functions `getYurthubKubeletConf` `getKubeletSvcBackup`

it may not be related to object `ConvertEdgeNodeOptions`, so keep it as a pure function?

keep it can still make some sense. what do you think?

Reply:
Agree, can change it to a pure func

Reply:
what about `getYurthubKubeletConf` `getKubeletSvcBackup`
change too ?

Reply:
Ok. And func `getYurthubConf()` in `pkg/yurtctl/cmd/revert/edgenode.go` can change together.

Reply:
after a second thought.

the root problem may be : 
`
      func (r *RevertEdgeNodeOptions) RunRevertEdgeNode() (err error) {
`
this core function related to `RevertEdgeNodeOptions`, so in the function, we use r.xx everywhere 

the better way may be：
1.  pure function   ` func RunRevertEdgeNode(o RevertEdgeNodeOptions)`
2. call it in 
```
	r := NewRevertEdgeNodeOptions()
        ... 
        Run: func(cmd *cobra.Command, _ []string) {
                         ...
			RunRevertEdgeNode(r)
		},
```

what do you think ？

Reply:
Both implementation methods are OK, but I prefer to keep the original method because it doesn't affect anything.
Just change these functions to public and pure functions so that convert and revert can be called at the same time.

Reply:
done.
but keep them as privite.